### DISCUSSION TITLE
=。-  多多提意见

### DISCUSSION THREAD

**Discussion by innocces** [innocces/taro-hooks#3] [Announcements]
_(:з」∠)_  需要建议

**Comment by Rainer-Yu** [comment]
个人拙见, vue示例里不要用useTaroEffect这种东西, 可以支持但没必要统一成类react的方式写示例代码, 和vue严重割裂

**Reply by innocces** [reply]
=。-  这个只是为了兼容多端写了一套这样的hook。觉得写了就可以放出来让大家用顺便验证一下。有歧义的话我这边后面隐藏掉。
